### PR TITLE
Add query nft trades labels

### PR DESCRIPTION
**Pull request by soispoke**
Brief comments on the purpose of your changes:
This PR aims to give an example of adding query labels to the spellbook repo

*For Dune Engine V2*
I've checked that:
General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [x] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/labels/nft/labels_nft_traders_transactions.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            THEN 'Top 10% NFT Trader (Transaction)'
+         WHEN ((ROW_NUMBER() OVER(ORDER BY COUNT(tx_hash) DESC)) / total_count * 100) <= 5 
+              AND ((ROW_NUMBER() OVER(ORDER BY COUNT(tx_hash) DESC)) / total_count * 100) > 1 
+            THEN 'Top 5% NFT Trader (Transactions)'
+         WHEN ((ROW_NUMBER() OVER(ORDER BY COUNT(tx_hash) DESC)) / total_count * 100) <= 1 
+            THEN 'Top 1% NFT Trader (Transactions)' END AS name,
+    'nft' AS category,
+    'soispoke' AS contributor,
+    'query' AS source,
+    timestamp('2022-09-24') as created_at,
```
[the remark this answers is not in the document]
Comment (soispoke) [reply models/labels/nft/labels_nft_traders_transactions.sql]: oh right thanks haha, this is fixed now 👍

File: models/labels/labels_all_schema.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      - name: category
+        description: "Label category"
+      - name: contributor
+        description: "Wizard(s) contributing to labels"
+      - name: source
+        description: "How were labels generated (could be static or query)"
+      - name: created_at
+        description: "When were labels created"
+      - name: updated_at
+        description: "When were labels updated for the last time ?"
```
[the remark this answers is not in the document]
Comment (soispoke) [reply models/labels/labels_all_schema.yml:29]: sounds good, done

File: models/labels/labels_all.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{{ config(
+    alias = 'all',
+    materialized = 'table',
+    file_format = 'delta')
```
[the remark this answers is not in the document]
Comment (soispoke) [reply models/labels/labels_all.sql:4]: Yes, I initially didn't want to expose the labels yet, until we have added a couple more queries/static labels. I'll try to add the alter statement in a post_hook in # 1474 if that's alright ?

**Review by jeff-dude — APPROVED** [review]
this builds fine, but a bit slow to query the end result in 'all' table, at least when trying to do select * on it. not a huge surprise as it's on large volume of nft trades data. 

also curious how you proceed with the few PRs all for labels? merge one PR, then merge main into other branch(s) and increment from there, considering they all need to union into the 'all' table?

**Comment by soispoke** [comment]
@jeff-dude thanks, yes exactly the goal was to give examples for static labels and query labels for it to be easier to review, but now I'll merge this, and then merge main into the bigger Labels WIP PR #1474. If you guys can review/approve that one when you have time it would be awesome

**Pull request merged**